### ISSUE TITLE
Add z-axis position in the 2D viewer

### ISSUE THREAD

**Issue by neuroBazzi** [MetaCell/salk-interactive-atlas#290]
From the client
_The 2D-viewer shows the Z-axis position but not the cervical level (C1-C8) as you
scroll through the cord. Even though it is possible to select it from “Custom View”, we
want the cervical level to be displayed on the 2D plot._

Here they would like to see the cervical level, so for example C2 if I'm in C2. They really like scrolling and they would like to see which level they are in while they scroll:

<img width="570" alt="Screenshot 2023-09-08 at 08 36 53" src="https://github.com/MetaCell/salk-interactive-atlas/assets/57769656/e5971170-5fa3-4fb2-b378-b92640235527">

**Comment by afonsobspinto** [comment]
I think we can add a legend somewhere, maybe on the right side?

If that's acceptable, and considering that I remember not having a good experience with the 2D viewer:
<details>
  <summary> I would cautiously estimate something around</summary>
4 hours  
</details>

**Comment by neuroBazzi** [comment]
Yes, I'd ask Sya to chime in, but I think so too. That number sounds good to me

**Comment by afonsobspinto** [comment]
https://www.figma.com/file/sWj9a09BlfZknccnGqTLnV/Salk-Mouse-Cord-Atlas?type=design&node-id=2678-15396&mode=design&t=rQ4EnqEdPC8iwQDR-4
![image](https://github.com/MetaCell/salk-interactive-atlas/assets/19196034/807759ed-84e9-4130-b934-b6131a581180)
